### PR TITLE
Fix text_classification examples multi_label.

### PR DESCRIPTION
**Pull request by songzy12**
1. code fix: dataset_name -> file_name in predict.py
2. add default value for params_path in predict.py
3. remove misleading comments in train.py: we have no demo for ernie in the multi_label example.
4. remove unused imports.
5. run yapf formatter on touch files.

### CODE REVIEW

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Comment by ZeyuChen** [comment]
@linjieccc Please review the code again.

**Comment by songzy12** [comment]
@ZeyuChen Thanks for the quick response.

May I ask what auto formatter we are using or is recommended for the PaddleNLP project?

I have tried autopep8 and yapf, it seems both of them would introduce irrelevant code style changes.
Or is it recommended we do not run autoformatter and just keep the untouched code as before?

File: examples/text_classification/multi_label/train.py [hunk 1]
```diff
@@ -93,12 +96,7 @@ def do_train():
     train_ds = load_dataset(read_custom_data, filename=os.path.join(
         args.data_path, file_name), is_test=False, lazy=False)
 
-    # If you wanna use ernie pretrained model,
```
Comment (linjieccc) [inline examples/text_classification/multi_label/train.py:96]: @songzy12 Thanks for your contribution! Since the example of multi-label text classification follows the strategy of fine-tuning from the pretrained model. This comment here tells the users that they can also use the other pretrained models provided by PaddleNLP Transformer API.

**Comment by ZeyuChen** [comment]
> @ZeyuChen Thanks for the quick response.
> 
> May I ask what auto formatter we are using or is recommended for the PaddleNLP project?
> 
> I have tried autopep8 and yapf, it seems both of them would introduce irrelevant code style changes.
> Or is it recommended we do not run autoformatter and just keep the untouched code as before?

We use pre-commit and PEP8 standard for coding style checking.

Please refer to this pre-commit config:  https://github.com/PaddlePaddle/PaddleNLP/blob/develop/.pre-commit-config.yaml

https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/10_contribution/local_dev_guide_cn.html#pre-commit

And also check this pre-commit usage in Paddle

@songzy12

File: examples/text_classification/multi_label/train.py (hunk 1 above)
Comment (songzy12) [reply examples/text_classification/multi_label/train.py:96]: Thanks for the review!

Regarding the pretrained model example, I plan to use an extra flag to select which model  to use like in the rnn example [1] rather than keep them in the comment [2]. Does this plan seems good?

[1] https://github.com/PaddlePaddle/PaddleNLP/blob/af82e03bbfc50ee4748a90f914c347f1e3f6ddcc/examples/text_classification/rnn/train.py#L104-L159
[2] https://github.com/PaddlePaddle/PaddleNLP/blob/af82e03bbfc50ee4748a90f914c347f1e3f6ddcc/examples/text_classification/pretrained_models/train.py#L151-L164

**Comment by songzy12** [comment]
@ZeyuChen Thanks for the reference!

**Comment by chenxiaozeng** [comment]
@songzy12 Thanks for your contribution! Hope to have futrher communication with you.
Weclome to email us by paddlenlp@baidu.com or leave behind your contact method. Thank you!

File: examples/text_classification/multi_label/train.py (hunk 1 above)
Comment (linjieccc) [reply examples/text_classification/multi_label/train.py:96]: Thanks a lot for the advise! 

For Simplicity, the pretrained model is default to BERT in this example. 
And we have updated the usage instruction of pretrained models in multi-label text classification: https://github.com/PaddlePaddle/PaddleNLP/pull/979
